### PR TITLE
feat(MembershipsAdapter): add addMembersToRoom method

### PR DESCRIPTION
**Pull request by harsh789rai**
add an addMembersToRoom method to MembershipsAdapter to add a member in the room for the given room id.

### CODE REVIEW

File: src/MembershipsAdapter.js [hunk 1]
```diff
@@ -59,4 +59,20 @@ export default class MembershipsAdapter extends WebexAdapter {
       new Error('getMembersFromDestination(destinationID, destinationType) must be defined in MembershipsAdapter'),
     );
   }
+
+  /**
+   * Add members to room for the given room id.Returns an observable
```
Comment (kishorepmr) [inline src/MembershipsAdapter.js:64]: * Add members to the given room. 

Move the **Returns an observable that emits the Member array.** to next line

File: src/MembershipsAdapter.js [hunk 2]
```diff
@@ -59,4 +59,20 @@ export default class MembershipsAdapter extends WebexAdapter {
...
+   * that emits the Member array with the data after adding members
+   * to room.
+   * Observable will complete after one emission
+   *
+   * @param {string} roomId Room id in which members are to be added
```
Comment (kishorepmr) [inline src/MembershipsAdapter.js:69]: roomId -> roomID

and description also **ID of the room where members are to be added.**

File: src/MembershipsAdapter.js [hunk 3]
```diff
@@ -59,4 +59,20 @@ export default class MembershipsAdapter extends WebexAdapter {
...
+   * @param {Array} roomMembers Emails of person to be added in room for the given room id
```
Comment (kishorepmr) [inline src/MembershipsAdapter.js:70]: @harsh789rai the create membership of JS SDK accepts email as well as person ID as well.

I think the parameter should be Array of Member object or Person Object.

File: src/MembershipsAdapter.js [hunk 4]
```diff
@@ -59,4 +59,20 @@ export default class MembershipsAdapter extends WebexAdapter {
...
+   * @param {bool} isModerator boolean to add member as a moderator in the room
+   * @returns {Observable<Member[]>} Observable that emits data after members added to room
```
Comment (kishorepmr) [inline src/MembershipsAdapter.js:72]: Observable stream that emits member data.

**Review by kishorepmr — CHANGES_REQUESTED** [review]

File: src/MembershipsAdapter.js (hunk 4 above)
Comment (harsh789rai) [reply src/MembershipsAdapter.js]: All the changes addressed

**Comment by lalli-flores** [comment]
@harsh789rai , @kishorepmr Team added similar definitions in https://github.com/webex/component-adapter-interfaces/pull/93

**Pull request closed without merging**